config/environments/console.js

```diff
@@ -113,5 +113,14 @@ export function getConfig() {
     options: {}
   };
 
+  config.oauth = {
+    facebookClientId:     '',
+    facebookClientSecret: '',
+    googleClientId:       '',
+    googleClientSecret:   '',
+    githubClientId:       '',
+    githubClientSecret:   '',
+  };
+
   return config;
```

Comment:
There change should also be applied to the 'config/environments/test.js' file

Reply:
*Done.*